lib/components/viewers/route-viewer.js

```diff
@@ -66,52 +71,30 @@ class RouteViewer extends Component {
   onSearchChange = (event) => {
     const { target } = event
     const { value } = target
+    // Search field state is duplicated to ensure there is no
+    // input lag while the redux store updates
```

Comment:
I feel like there really shouldn't be much lag there... maybe there are some perf improvements we should target.

Reply:
Fixed this issue in e886cfc75dde3dbec1844daa3cd88c51a0750e23 (line 56 of route-viewer.js) and removed react state in 428ff8bd42e0757e6967d56f15b19efe0d8f63b3. Still runs smooth!